src/wallet/rpcwallet.cpp

```diff
@@ -3323,10 +3323,16 @@ static UniValue bumpfee(const JSONRPCRequest& request)
             "         \"CONSERVATIVE\""},
                         },
                         "options"},
+                    {"add_to_wallet", RPCArg::Type::BOOL, /* default */ "true", "When false, returns the fee-bumped\n"
```

Comment:
Suggest making the option leave it unsigned too. User can always pass it back in to a signing RPC.

Reply:
meh, ~0